injected/src/features/google-rejected.js

```diff
@@ -21,6 +21,12 @@ export default class GoogleRejected extends ContentFeature {
             if ('adAuctionComponents' in Navigator.prototype) {
                 delete Navigator.prototype.adAuctionComponents
             }
+            // https://github.com/WICG/privacy-preserving-ads/blob/main/API%20Details.md
+            if (this.getFeatureSetting('getInterestGroupAdAuctionData')) {
```

Comment:
Why put it behind additional flag?

Reply:
Good practice really. In case we want to enable the API on a particular website, additionally there's higher risk as we've not seen it in the wild yet.